code/modules/clothing/head/berets.dm

```diff
@@ -247,3 +247,12 @@
 	desc = "A black beret adorned with the crest of an ERT detachment. Worn by commanders of NanoTrasen response teams."
 	icon_state = "centcomcaptain"
 	item_state = "centcomcaptain"
+
+// Factional
+
+/obj/item/clothing/head/beret/sapk
+	name = "peacekeeper beret"
+	desc = "A sky-blue Army beret with a white trim, proudly displaying a golden emblem of the Alliance."
+	desc_extended = "While a common sight in the Alliance's more rebellious or devastated worlds such as Mars, the Army peacekeeper beret became a potent symbol of Sol's triumph in the civil war. Nowhere else in the Spur are the blue berets more omnipresent than the former capital of the Restoration Front - Lycoris."
```

Comment:
1.

Searching "peacekeeper" on the wiki just shows me peacekeeper mandate (biesel faction), so it's just going to be confusing if this beret is referencing a similarly named solarian faction that's not actually in lore. There shouldn't be lore items in the loadout that aren't connected to actual lore on the wiki. Lycoris doesn't even have its own wiki page and its only referenced by name in a couple of other pages.

2.

Why would any crew wear this on horizon? Why would it be allowed by SCC? Especially with the bit of `the Army peacekeeper beret became a potent symbol of Sol's triumph in the civil war`. Biesel and corporations don't really like sol, and this is a beret of a military force that's (I assume) still active.

Reply:
Also generally not a fan of more and more military gear or uniforms being added to the loadout.

Reply:
(oops accidentally requested changes - meant to just post a comment)

Reply:
> but the problem is that it doesn't really match the color scheme of any of the existing Solarian clothes in sol_uniforms.dmi. It looks like a PMCG beret more than anything else.

It utilizes colors of the Solarian flag, white, blue, gold. 

![image](https://github.com/user-attachments/assets/2bbb89d9-315e-4563-b523-4ad47fd29ae3)

It is absolutely true that this is the odd one out- but that's on purpose. It's supposed to stand out and be recognizable, as well as add Sol's chief colors to their available military's color scheme. Currently what you have is _grey, olive green, black and white, red,_ which are very common colors throughout any uniform in the setting.

> Searching "peacekeeper" on the wiki just shows me peacekeeper mandate (biesel faction), so it's just going to be confusing if this beret is referencing a similarly named solarian faction that's not actually in lore. There shouldn't be lore items in the loadout that aren't connected to actual lore on the wiki. Lycoris doesn't even have its own wiki page and its only referenced by name in a couple of other pages.

![image](https://github.com/user-attachments/assets/094af2d3-cefa-4e03-bda8-f880b6673147)

![image](https://github.com/user-attachments/assets/1fa363ca-b10b-4170-97ba-4fccfdfec003)

https://forums.aurorastation.org/topic/2816-sol-alliance-news-network/?do=findComment&comment=177608

Peacekeeping isn't a term exclusive to Biesel, and has been used to reference Solarian forces in lore pages. The arc is still ongoing and as such not everything related to it will be present on the wiki.

I've discussed this with Schwann and he told me it makes sense to be added.

I can rename it to 'Solarian Army Beret', and leave mention of it's use by Solarian peacekeepers in the extended description if that's preferable to how it is now.

> Why would any crew wear this on horizon? Why would it be allowed by SCC? Especially with the bit of the Army peacekeeper beret became a potent symbol of Sol's triumph in the civil war. Biesel and corporations don't really like sol, and this is a beret of a military force that's (I assume) still active.

The SCC already allows people to display not only the Solarian flag in their departments, but the flags of Himeo and Pluto, which are far more overtly hostile to the SCC. Himeo outright executes it's people who affiliate with Hephaestus, yet not only are Himean voidsuit modkits allowed, you can also wear their flagpatch on your person.

Same with Pluto, you can be a card carrying member of a communist party, wear it's flagpatch, display it's flag, you can visually be a literal communist.

I don't see how a beret belonging to an occupational / garrison force that has never come into conflict with the SCC would push them to forbid it over things I've mentioned above.

Reply:
> The SCC already allows people to display not only the Solarian flag in their departments

> flags of Himeo and Pluto

A nation's flag is VERY different from a beret of an active military force.